src/lib/logical_query_plan/stored_table_node.hpp

```diff
@@ -43,8 +43,8 @@ class StoredTableNode : public EnableMakeForLQPNode<StoredTableNode>, public Abs
   std::vector<std::shared_ptr<AbstractExpression>> output_expressions() const override;
   bool is_column_nullable(const ColumnID column_id) const override;
 
-  // Generates unique constraints from table's key constraints and pays respect to pruned columns.
-  std::shared_ptr<LQPUniqueConstraints> unique_constraints() const override;
+  // Generates unique column combinations from table's key constraints and pays respect to pruned columns.
```

Comment:
"a table's"?

Reply:
Can we reformulate "pays respect"?

Reply:
`Generates unique column combinations from a table's key constraints. Drops UCCs that include pruned columns.`?

Reply:
Sound good.